bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -277,6 +277,27 @@
 		<config-description-ref uri="thing-type:avmfritz:fritzdevice"/>
 	</thing-type>
 
+	<thing-type id="HAN_FUN_BLIND">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="fritzbox"/>
+			<bridge-type-ref id="FRITZ_Powerline_546E_Solo"/>
+		</supported-bridge-type-refs>
+
+		<label>HAN-FUN Blind</label>
+		<description>HAN-FUN blind (e.g. RolloTron DECT 1213)</description>
+
+		<channels>
+			<channel id="rollershutter" typeId="rollershutter">
+				<label>Rollershutter control</label>
+				<description>Controls the rollershutter and displays its opening level in percent</description>
```

Comment:
"label" and "description" should be moved into the channel type definition. Please use CamelCase for labels.

Reply:
Done.